ajax/libs/vivid/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "vivid",
```

Comment:
Not sure that which name is better, `vivid-icons` or `vivid`?
Because other CDNs use `vivid-icons` as this package name, ref: https://www.npmjs.com/package/vivid-icons#link-from-jsdelivr-recommended-or-unpkg-cdn, also the package managers use `vivid-icons` name too.
Thanks